app-header-layout/app-header-layout.html

```diff
@@ -147,6 +149,32 @@
         this._updateScroller();
       },
 
+      // Resizes app-header-layout to the height of the viewport and translates the content
+      // up to "simulate" disabled document scrolling on iOS.
+      simulateDisabledScroll: function(disabled) {
+        if (disabled) {
+          if (this.hasScrollingRegion) {
+            this.$.contentContainer.style.webkitOverflowScrolling = 'auto';
+          } else {
+            this._savedPageYOffset = window.pageYOffset;
```

Comment:
Note to self: need to add docs around this, especially why using just `webkitOverflowScrolling` works for element scrollers.